src/gettingstarted/own-code/push-project.md

```diff
@@ -0,0 +1,49 @@
+# Import your own code
+
+## Build, Deploy, Done!
+
+With your configuration files complete, all that's left is to commit the changes and push to Platform.sh.
+
+<asciinema-player src="/videos/asciinema/first-push.cast" preload=1 autoplay=1 loop=1></asciinema-player>
```

Comment:
The SSH key validation step in here is just distracting. Can it be recorded without that?

Reply:
In all of the Asciinema videos we did for the demo slides, we converted "us typing stuff" to a single instant command with a 2 second delay after it so that users could read it.  Should we do the same on these recordings?  We aren't as pressed for time, but we still don't want users to just be waiting for us to type things.  I'm open to discussion here.